lib/stdlib/test/lists_SUITE.erl

```diff
@@ -2743,6 +2743,13 @@ hof(Config) when is_list(Config) ->
 
     ok.
 
+%% Test lists:enumerate/1 and lists:enumerate/2
+enumerate(Config) when is_list(Config) ->
+    [] = lists:enumerate([]),
+    [] = lists:enumerate(10, []),
+    [{1,a},{2,b},{3,c}] = lists:enumerate([a,b,c]),
+    [{10,a},{11,b},{12,c}] = lists:enumerate(10, [a,b,c])
```

Comment:
This can't compile, there is a missing dot here,
and the test case should return ok.
You might want to run the test-code :-)

Add some tests for bad input as well while you are at it.

Reply:
You can run the lists suite with: `make test ARGS='-suite lists_SUITE'`

Reply:
Uh, I got fooled by the passing checks in the PR. Any idea why they didn't catch it?

Reply:
The github actions checks only compile and run a small subset of tests. The lists SUITE is not one of those.